tests/contracts/SimpleWallet.sol

```diff
@@ -34,6 +34,12 @@ contract SimpleWallet is IAccount {
         }
         bytes2 sig = bytes2(userOp.signature);
         require(sig != 0xdead, "testWallet: dead signature");
-        return sig == 0xdeaf ? 1 : 0;
+        if (sig == 0xdeaf) return 1;
+        if (userOp.signature.length == 64) {
+            (uint48 validUntil, uint48 validAfter) = abi.decode(userOp.signature,(uint48, uint48));
+            return (uint256(validUntil) << 160) | (uint256(validAfter) << (160 + 48));
```

Comment:
this is what we have `_packValidationData()` helper method for...

Reply:
Fixed